net-misc/tigervnc/tigervnc-1.6.0-r2.ebuild

```diff
@@ -145,7 +150,7 @@ src_compile() {
 	if use server; then
 		# deps of the vnc module and the module itself
 		local d subdirs=(
-			fb xfixes Xext dbe glx randr render damageext miext Xi xkb
+			fb xfixes Xext dbe $(usex opengl glx "") randr render damageext miext Xi xkb
```

Comment:
In case of USE=-opengl, we shouldn't be building anything inside `glx` directory.
The order of directories is also important, so the code ensures that `glx` is there in case of USE=opengl and not in case of USE=-opengl.